singularity-eos/eos/eos.hpp

```diff
@@ -1015,7 +1000,7 @@ class StellarCollapse : EosBase<StellarCollapse> {
 #ifdef SINGULARITY_USE_EOSPAC
 // Only really works in serial
 // Not really supported on device
-class EOSPAC : EosBase<EOSPAC> {
+class EOSPAC : public EosBase<EOSPAC> {
```

Comment:
Should eospac be inheriting from the base class since it's not going to be using the base class helper functions?

Reply:
It shouldn't and I plan to change that. I added that as a quick workaround to get things to compile at this stage while I work on getting EOSPAC itself working in a separate way

Reply:
So in short, I plan to keep the inheritance but only use the `using` statements when needed

Reply:
But that reminds me that I need to actually change the scalar version of the member function in the `Variant` class... whoops 😅